README.md

```diff
@@ -73,215 +176,15 @@ This feature is available once you upgrade to version 2.6.4 and above. Follow th
-
-The code example below shows how to enable tracing. For more details, see [here](https://cloud.google.com/community/tutorials/bigtable-oc).
-
-##### Maven Setup
-
-If you are _not_ using the shaded Bigtable HBase Client artifact, you need to define the OpenCensus dependencies.
-
-[//]: # ({x-version-update-start:bigtable-client-parent:released})
-```xml
-<!-- OpenCensus dependencies -->
-<dependency>
-    <groupId>com.google.cloud.bigtable</groupId>
-    <artifactId>bigtable-hbase-1.x</artifactId>
-    <version>2.6.5</version>
-</dependency>
-```
-[//]: # ({x-version-update-end})
-```xml
-<dependency>
-    <groupId>io.opencensus</groupId>
-    <artifactId>opencensus-impl</artifactId>
-    <version>0.31.1</version>
-    <scope>runtime</scope>
-</dependency>
-<dependency>
-    <groupId>io.opencensus</groupId>
-    <artifactId>opencensus-exporter-trace-stackdriver</artifactId>
-    <version>0.31.1</version>
-    <exclusions>
-        <exclusion>
-            <groupId>io.grpc</groupId>
-            <artifactId>*</artifactId>
-        </exclusion>
-        <exclusion>
-            <groupId>com.google.auth</groupId>
-            <artifactId>*</artifactId>
-        </exclusion>
-    </exclusions>
-</dependency>
-```
-
-If you _are_ using the shaded Bigtable HBase Client artifact, then the OpenCensus dependencies are embedded in the
-shaded artifact; i.e. nothing additional for you to do.
-
-[//]: # ({x-version-update-start:bigtable-client-parent:released})
-```xml
-<!-- OpenCensus dependencies -->
-<dependency>
-    <groupId>com.google.cloud.bigtable</groupId>
-    <artifactId>bigtable-hbase-1.x-shaded</artifactId>
-    <version>2.6.5</version>
-</dependency>
-```
-[//]: # ({x-version-update-end})
+## Client Metrics
 
-##### Java Example
 
 ```java
-// For the non-shaded client, remove the package prefix "com.google.bigtable.repackaged."
-import com.google.bigtable.repackaged.io.opencensus.exporter.trace.stackdriver.StackdriverTraceConfiguration;
-import com.google.bigtable.repackaged.io.opencensus.exporter.trace.stackdriver.StackdriverTraceExporter;
-import com.google.bigtable.repackaged.io.opencensus.trace.Tracing;
-import com.google.bigtable.repackaged.io.opencensus.trace.samplers.Samplers;
-
-import java.io.IOException;
-
-public class OpenCensusExample {
-    String projectId = "your-project-id";
-
-    void setupTracing() throws Exception {
-        // Setup tracing.
-        StackdriverTraceExporter.createAndRegister(
-                StackdriverTraceConfiguration.builder()
-                        .setProjectId(projectId)
-                        .build()
-        );
-        Tracing.getTraceConfig().updateActiveTraceParams(
-                Tracing.getTraceConfig().getActiveTraceParams().toBuilder()
-                        // Adjust the sampling rate as you see fit.
-                        .setSampler(Samplers.probabilitySampler(0.01))
-                        .build()
-        );
-    }
-}
+BigtableConfiguration.enableBuiltinMetrics();
```

Comment:
I would add something here, "to enable client side metrics, add:"

Reply:
done